src/templates/footer/partials/apps-and-social-links.hbs

```diff
@@ -31,22 +31,8 @@
 
 <div class="g-col g-span3 c-footer-feedback is-invisible" data-gtm-feedback>
     <h2 class="c-footer-heading">{{ i18n "feedback" }}</h2>
-    <div class="g">
-        <div class="g-col g-span2">
-            <div class="c-footer-feedbackLogo">
-                <svg width="44" height="32" viewBox="0 0 44 32">
-                    <g id="o-feedbackLogo" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-                        <path d="M31.1433088,26.7702396 L31.9029017,26.7702396 L34.0550816,28.8968474 L36.7136567,31.5238335 C36.8402556,31.6489281 36.9668544,31.7740227 37.0934532,31.7740227 C37.220052,31.7740227 37.3466508,31.8991173 37.4732496,31.8991173 C37.5998485,31.8991173 37.8530461,31.7740227 37.9796449,31.6489281 C38.1062437,31.398739 38.2328425,31.1485498 38.2328425,30.7732661 L38.2328425,25.0189155 C41.1446153,23.392686 43.0435976,20.8907945 43.0435976,18.0136192 C43.0435976,13.2600252 37.7264473,9.25699874 31.1433088,9.25699874 C30.2571171,9.25699874 29.3709254,9.38209332 28.4847337,9.50718789 L27.3453443,15.7619168 L20.3824094,19.2645649 L19.2430201,18.0136192 C19.2430201,22.7672131 24.5601704,26.7702396 31.1433088,26.7702396 Z" id="Shape" fill="#535353"></path>
-                        <path d="M14.9386603,0 C6.70973727,0 0.126598816,4.87868852 0.126598816,10.8832282 C0.126598816,14.510971 2.53197633,17.6383354 6.203342,19.6398487 L6.203342,26.8953342 C6.203342,27.2706179 6.32994082,27.6459016 6.45653963,27.8960908 C6.58313845,28.0211854 6.83633608,28.1462799 7.08953372,28.1462799 C7.21613253,28.1462799 7.46933016,28.0211854 7.59592898,28.0211854 C7.7225278,27.8960908 7.97572543,27.8960908 8.10232425,27.6459016 L11.3938935,24.3934426 L14.0524686,21.7664565 L15.0652591,21.7664565 C23.2941822,21.7664565 29.8773207,16.887768 29.8773207,10.8832282 C29.8773207,4.87868852 23.1675834,0 14.9386603,0 L14.9386603,0 Z M14.9386603,18.3889029 C8.73531833,18.3889029 3.54476686,14.8862547 3.54476686,11.0083228 C3.54476686,7.00529634 8.73531833,3.50264817 14.9386603,3.50264817 C21.1420023,3.50264817 26.3325538,7.00529634 26.3325538,10.8832282 C26.4591526,15.0113493 21.1420023,18.3889029 14.9386603,18.3889029 L14.9386603,18.3889029 Z" id="Shape" fill="#535353"></path>
-                    </g>
-                </svg>
-            </div>
-        </div>
-        <div class="g-col g-span10">
-            <p class="c-footer-feedbackText">
-                {{ i18n "improveOurWebsite" }}
-                <a href="#">{{ i18n "sendFeedback" }}</a>
-            </p>
-        </div>
-    </div>
+    <p class="c-footer-feedbackText">
+        {{ i18n "improveOurWebsite" }}
+        <a href="#" class="o-link--bold o-link--full">{{ i18n "sendFeedback" }}</a>
```

Comment:
This should probably be a button as it doesn't actually have a link tied to it. Maybe should be done in a different PR though as looks like you didn't change this as part of these updates.

Reply:
I can do it here as it is a small change

Reply:
As discussed, I will rise separate jira for this, because gtm use a tag for click event